examples/sycl/pvc/pvc_gemm_with_epilogue_lincombdeeltact.cpp

```diff
@@ -0,0 +1,452 @@
+        );
+
+    syclcompat::wait();
+    cutlass::reference::device::BlockElementwiseOp<sum_vals>(
+      block_ref_D.get(), block_ref_D.get(), block_Aux.get(), block_D.size());
+    syclcompat::wait();
+
+    // Check if output from CUTLASS kernel and reference kernel are equal or not
+    bool passed = cutlass::reference::device::BlockCompareEqual(
+      block_ref_D.get(), block_D.get(), block_D.size());
+
+    if (0 and !passed){
+      print_device_tensor(block_D, M, N, L, "Actual");
+      print_device_tensor(block_ref_D, M, N, L, "Reference");
+      cutlass::reference::device::BlockElementwiseOp<diff_vals>(block_ref_D.get(), block_ref_D.get(), block_D.get(), block_D.size());
+      syclcompat::wait();
+      print_device_tensor(block_ref_D, M, N, L, "difference");
+    }
+
+    return passed;
+  }
+
+  /// Initialize operands to be used in the GEMM and reference GEMM
+  void initialize(const ProblemShapeType& problem_size) {
+    auto problem_shape_MNKL = cute::append<4>(problem_size, 1);
+    auto [M, N, K, L] = problem_shape_MNKL;
+
+    stride_A = cutlass::make_cute_packed_stride(StrideA{}, cute::make_shape(M, K, L));
+    stride_B = cutlass::make_cute_packed_stride(StrideB{}, cute::make_shape(N, K, L));
+    stride_C = cutlass::make_cute_packed_stride(StrideC{}, cute::make_shape(M, N, L));
+    stride_D = cutlass::make_cute_packed_stride(StrideD{}, cute::make_shape(M, N, L));
+
+    block_A.reset(M * K * L);
+    block_B.reset(K * N * L);
+    block_C.reset(M * N * L);
+    block_D.reset(M * N * L);
+    block_ref_D.reset(M * N * L);
+    block_Aux.reset(M * N * L);
+
+    initialize_block(block_A, seed + 2023);
+    initialize_block(block_B, seed + 2022);
+    initialize_block(block_C, seed + 2021);
+    initialize_block(block_Aux, seed + 2020);
+  }
+
+  void run(const Options& options, const cutlass::KernelHardwareInfo& hw_info) {
+    ProblemShapeType problem_size = ProblemShapeType{options.m, options.n, options.k, options.l};
+
+    initialize(problem_size);
+
+    using EpilogueArguments = typename Gemm::GemmKernel::EpilogueArguments;
+    // TODO need to inject the aux argument here
+    EpilogueArguments epilogue_arguments{
+      {options.alpha, options.beta}, block_C.get(), stride_C, block_D.get(), stride_D};
+    epilogue_arguments.thread.aux_ptr = block_Aux.get();
+    epilogue_arguments.thread.dAux = cutlass::make_cute_packed_stride(StrideD{}, cute::make_shape(options.m, options.n, options.l));
+
+    typename Gemm::GemmKernel::Arguments arguments{
+      cutlass::gemm::GemmUniversalMode::kGemm, // mode
+      problem_size, // problem_shape
+      {block_A.get(), stride_A, block_B.get(), stride_B}, // mainloop
+      epilogue_arguments, // epilogue
+      hw_info // hw_info
+              // scheduler
+    };
+
+    Gemm gemm_op;
+
+    size_t workspace_size = Gemm::get_workspace_size(arguments);
+    cutlass::device_memory::allocation<uint8_t> workspace(workspace_size);
+
+    gemm_op.can_implement(arguments);
```

Comment:
Does calling this and ignoring return value make sense?

Reply:
Nope! I've added that now.